xml/System/Type.xml

```diff
@@ -5218,7 +5218,7 @@ The following <xref:System.Reflection.BindingFlags> modifier flags can be used t
   
 See <xref:System.Reflection.BindingFlags?displayProperty=nameWithType> for more information.  
 
-The <xref:System.Type.GetMembers%2A> method does not return members in a particular order, such as alphabetical or declaration order. Your code must not depend on the order in which members are returned, because that order varies.  
+In .NET 6 and below versions the <xref:System.Type.GetMembers%2A> method does not return members in a particular order, such as alphabetical or declaration order. Your code must not depend on the order in which members are returned, because that order varies. However, starting with .NET 7, the ordering is deterministic based upon the metadata ordering in the assembly.
```

Comment:
```suggestion
In .NET 6 and earlier versions, the <xref:System.Type.GetMembers%2A> method does not return members in a particular order, such as alphabetical or declaration order. Your code must not depend on the order in which members are returned, because that order varies. However, starting with .NET 7, the ordering is deterministic based upon the metadata ordering in the assembly.
```